docs/partials/_results-api-env-vars.mdx

```diff
@@ -0,0 +1,163 @@
+The `@cypress/extract-cloud-results` helper cross-references some environment variables from where it is executed with ones that were present when a Cypress Cloud run was recorded. This allows for automatically detecting the correct Cloud run when the Results API is invoked from the same CI context as a given run (as is the case in the above examples).
```

Comment:
This doc has a pretty big potential for becoming stale. It'd be great if we could somehow reference code to prevent this.